ee/agent/hardware_identifiers_other.go

```diff
@@ -0,0 +1,16 @@
+//go:build !windows
+// +build !windows
+
+package agent
+
+import (
+	"context"
+
+	"github.com/kolide/launcher/ee/agent/types"
+)
+
+// currentMachineGuid is only implemented for windows, where the hardware_uuid
+// cannot be used as a stable identifier
+func currentMachineGuid(_ context.Context, _ types.Knapsack) (string, error) {
+	return "", nil
```

Comment:
unsure whether to return an empty string or some const value here for non-windows. empty string seems more correct to me, but will have the effect of causing every startup to rewrite this empty string: https://github.com/kolide/launcher/blob/66a737764cbe8c92fd03cf6eb1e49f26e33bdb5e/ee/agent/reset.go#L243-L248
the end result will still be what we want either way, and we could also change that logic to just not write out empty strings but maybe there's something I'm not considering? curious what others think here

Reply:
```if len(storedValue) == 0 && len(currentValue) > 0 {```

seems fine to me? Would be nice to not have the extra log spam, and I agree that an empty string seems better than a const